### ISSUE TITLE
Exception causes and suppressed exceptions aren't returned in SerializableError

### ISSUE THREAD

**Issue by SerialVelocity** [palantir/conjure-java-runtime#407]
Exception causes and suppressed exceptions aren't being returned. This makes it very difficult to use the returned stack traces because you end up with something like:
```
UncheckedIOException: An error occurred deleting a folder
   <stack trace here>
```
instead of:
```
UncheckedIOException: An error occurred deleting a folder
   <stack trace here>
Caused by: IOException: Folder was not empty
   <stack trace here>
```

**Comment by pnepywoda** [comment]
Even logging the exception message would help, if logging the stack trace is too much

**Comment by uschi2000** [comment]
The full request gets logged: https://github.com/palantir/http-remoting/blob/develop/jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java#L55

As long as the wire-format remains stable, we can add nested stacktraces.

**Comment by pnepywoda** [comment]
It was a witchcraft-specific issue, fixed in 0.37.2 (it now contains stack traces in logs)

**Comment by uschi2000** [comment]
no longer relevant
